### PR TITLE
[bsp][nxp][mcxa153] add spi1 driver entry

### PR DESCRIPTION
**Pull request by hywing**
## 拉取/合并请求描述：(PR description)

[

#### 为什么提交这份PR (why to submit this PR)

add spi1 driver entry for NXP MCXA153 Board

#### 你的解决方案是什么 (what is your solution)

add && modify spi driver files in `bsp\nxp\mcx\mcxa\Libraries\drivers`

#### 请提供验证的bsp和config (provide the config and bsp) 

- BSP: BSP_USING_SPI1

- .config: BSP_USING_SPI1

- action: https://github.com/hywing/rt-thread/commit/1f9009c5c8f8b277ade412bf0d98467381443f22

]

### 当前拉取/合并请求的状态 Intent for your PR

必须选择一项 Choose one (Mandatory):

- [ ] 本拉取/合并请求是一个草稿版本 This PR is for a code-review and is intended to get feedback
- [x] 本拉取/合并请求是一个成熟版本 This PR is mature, and ready to be integrated into the repo

### 代码质量 Code Quality：

我在这个拉取/合并请求中已经考虑了 As part of this pull request, I've considered the following:

- [x] 已经仔细查看过代码改动的对比 Already check the difference between PR and old code
- [x] 代码风格正确，包括缩进空格，命名及其他风格 Style guide is adhered to, including spacing, naming and other styles
- [x] 没有垃圾代码，代码尽量精简，不包含`#if 0`代码，不包含已经被注释了的代码 All redundant code is removed and cleaned up
- [x] 所有变更均有原因及合理的，并且不会影响到其他软件组件代码或BSP All modifications are justified and not affect other components or BSP
- [x] 对难懂代码均提供对应的注释 I've commented appropriately where code is tricky
- [x] 代码是高质量的 Code in this PR is of high quality
- [x] 已经使用[formatting](https://github.com/mysterywolf/formatting) 等源码格式化工具确保格式符合[RT-Thread代码规范](https://github.com/RT-Thread/rt-thread/blob/master/documentation/contribution_guide/coding_style_cn.md) This PR complies with [RT-Thread code specification](https://github.com/RT-Thread/rt-thread/blob/master/documentation/contribution_guide/coding_style_en.md)

### CODE REVIEW

**Comment by hywing** [comment]
代码已经格式化完毕，不知为啥会报错
image

File: bsp/nxp/mcx/mcxa/frdm-mcxa153/board/MCUX_Config/board/pin_mux.c [hunk 1]
```diff
@@ -53,6 +53,7 @@ void BOARD_InitPins(void)
 
     RESET_ReleasePeripheralReset(kCTIMER1_RST_SHIFT_RSTn);
     RESET_ReleasePeripheralReset(kLPSPI0_RST_SHIFT_RSTn);
+	RESET_ReleasePeripheralReset(kLPSPI1_RST_SHIFT_RSTn);
```
Comment (mysterywolf) [inline bsp/nxp/mcx/mcxa/frdm-mcxa153/board/MCUX_Config/board/pin_mux.c:56]: 这里不用tab 用4空格代替
```suggestion
    RESET_ReleasePeripheralReset(kLPSPI1_RST_SHIFT_RSTn);
```

File: bsp/nxp/mcx/mcxa/frdm-mcxa153/board/Kconfig [hunk 2]
```diff
@@ -53,6 +53,10 @@ menu "On-chip Peripheral Drivers"
             if BSP_USING_SPI
                 config BSP_USING_SPI0
                     bool "Enable SPI0"
+                    default n
+					
+				config BSP_USING_SPI1
```
Comment (mysterywolf) [inline bsp/nxp/mcx/mcxa/frdm-mcxa153/board/Kconfig:58]: 请使用4空格代替tab

**Review by mysterywolf — CHANGES_REQUESTED** [review]

File: bsp/nxp/mcx/mcxa/frdm-mcxa153/board/Kconfig [hunk 3]
```diff
@@ -55,7 +55,7 @@ menu "On-chip Peripheral Drivers"
                     bool "Enable SPI0"
                     default n
 					
-				config BSP_USING_SPI1
+		config BSP_USING_SPI1
```
Comment (mysterywolf) [inline bsp/nxp/mcx/mcxa/frdm-mcxa153/board/Kconfig:58]: 这里还是tab 不是4空格

File: bsp/nxp/mcx/mcxa/frdm-mcxa153/board/Kconfig [hunk 4]
```diff
@@ -54,6 +54,10 @@ menu "On-chip Peripheral Drivers"
                 config BSP_USING_SPI0
                     bool "Enable SPI0"
                     default n
+					
+	        config BSP_USING_SPI1
```
Comment (mysterywolf) [inline bsp/nxp/mcx/mcxa/frdm-mcxa153/board/Kconfig:58]:
```suggestion
                config BSP_USING_SPI1
```

**Review by mysterywolf — APPROVED** [review]

**Pull request merged**